cowprotocol/cowswap
# Updated ToC page

# Summary

This PR updates ToC page with a new version of ToC as defined here https://docs.google.com/document/d/1xxTUoCFYnk4O0Wodqfi-WwhYGg9tTm2ayOYHuXDliLg/edit

# To test
- go to `/terms-and-conditions` page
- make sure the content is correct as defined in the doc
